composer.json

```diff
@@ -25,12 +25,17 @@
         "symfony/http-kernel": "^2.3 || ^3.0"
     },
     "require-dev": {
+        "jms/di-extra-bundle": "^1.7",
         "knplabs/knp-menu-bundle": "^2.0",
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
         "sonata-project/admin-bundle": "^3.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
+    "conflict": {
+        "jms/di-extra-bundle": "<1.7.0"
+    },
     "suggest": {
+        "jms/di-extra-bundle": "Annotations for Block definition",
```

Comment:
@OskarStark I think this is the right way because this feature should be considered optional.